### DISCUSSION TITLE
Using SMS OTP With Email Provider Accounts

### DISCUSSION THREAD

**Discussion by no2pixel** [supabase/supabase#11655] [Questions]
Users sign up using email, but can add their phone for receiving OTP for easier login. This introduces a couple of problems, or perhaps misunderstandings of capabilities.

When a user adds their phone number, they receive a confirmation OTP by sms, that when verified with verifyOTP() with a 'phone_change' verification type, properly adds the phone number to the user, and future OTP can be sent either via the dashboard or supabase.js -- but it doesn't seem possible to actually log in using an SMS OTP for an account that originally signed up by email. Passing the email and the OTP results in an "Invalid or Expired" error, and signing in with the phone number and OTP results in a "user not found" error. 

Is it possible to mix SMS OTP and email login in this way? If not, is there a way to create a second identity with supabase.js without the service_key (the actual app is built on Webflow, so there is no server-side functions to be able to use the service_key without exposing it to client-side)?

**Comment by bombillazo** [comment]
Im interested in this as well, we may very well want to verify and allow users to login via email and phone.
